data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/master-update.fcc

```diff
@@ -10,7 +10,7 @@ storage:
       path: /etc/kubernetes/bootstrap-configs
     - local: tls/
       path: /etc/kubernetes/bootstrap-secrets
-    - local: etcd-bootstrap/bootstrap-manifests/secrets/
+    - local: etcd-bootstrap/etc-kubernetes/bootstrap-secrets/
```

Comment:
This is wrong.
The content should be:
```
[root@master1 ~]# ls /opt/openshift/etcd-bootstrap/bootstrap-manifests/secrets/
etcd-all-peer  etcd-all-serving
```
2021-03-04 13:40:15.983736 C | etcdmain: open /etc/ssl/etcd/etcd-all-certs/etcd-peer-test-infra-cluster-master-0.crt: no such file or directory
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_installer/4712/pull-ci-openshift-installer-master-e2e-metal-single-node-live-iso/1367452012081319936/artifacts/e2e-metal-single-node-live-iso/baremetalds-sno-gather/artifacts/build/ibip/installer-gather/log-bundle-20210304134143/bootstrap/containers/etcd-c77acea1d839c6b7ce945571ce64cf36ef8b26bea96134d8474931170ed8cd9e.log